hapi/src/api/user.api.js

```diff
@@ -44,7 +44,6 @@ const SET_NAME = `
 
 const getOne = async (where = {}) => {
   const { user } = await hasuraUtils.request(GET_ONE, { where })
-
```

Comment:
Why you delete that blank space?

Remember to add blank space after const or let declaration!